Classes/View Controllers/UIViewController+MenuDone.swift

```diff
@@ -10,13 +10,22 @@ import UIKit
 
 extension UIViewController {
 
-    func addMenuDoneButton() {
-        navigationItem.rightBarButtonItem = UIBarButtonItem(
-            barButtonSystemItem: .done,
-            target: self,
-            action: #selector(onMenuDone)
-        )
-        navigationItem.rightBarButtonItem?.tintColor = Styles.Colors.Gray.light.color
+    func addMenuDoneButton(left: Bool = false) {
+        if left {
+            navigationItem.leftBarButtonItem = UIBarButtonItem(
+                barButtonSystemItem: .done,
+                target: self,
+                action: #selector(onMenuDone)
+            )
+            navigationItem.leftBarButtonItem?.tintColor = Styles.Colors.Gray.light.color
+        } else {
+            navigationItem.rightBarButtonItem = UIBarButtonItem(
+                barButtonSystemItem: .done,
+                target: self,
+                action: #selector(onMenuDone)
+            )
+            navigationItem.rightBarButtonItem?.tintColor = Styles.Colors.Gray.light.color
```

Comment:
Lots of duped code here. Can we just capture which bar button item we're operating on and set everything once?